Geometry/HGCalCommonData/src/HGCalGeomRotation.cc

```diff
@@ -0,0 +1,96 @@
+#include "Geometry/HGCalCommonData/interface/HGCalGeomRotation.h"
+#include "FWCore/Utilities/interface/Exception.h"
+
+void HGCalGeomRotation::uvMappingFromSector0(WaferCentring waferCentring,
+                                             int& moduleU,
+                                             int& moduleV,
+                                             unsigned sector) const {
+  if (sector == 0) {
+    return;
+  }
+
+  int offset;
+
+  if (waferCentring == WaferCentred) {
+    offset = 0;
+  } else if (waferCentring == CornerCentredY) {
+    offset = -1;
+  } else if (waferCentring == CornerCentredMercedes) {
+    offset = 1;
+  } else {
+    throw cms::Exception("RotationException")
+        << "HGCalGeomRotation: WaferCentring must be one of: WaferCentred, CornerCentredY or CornerCentredMercedes";
+  }
+
+  int uPrime, vPrime;
+
+  if (sector == 1) {
+    uPrime = -moduleV + offset;
+    vPrime = moduleU - moduleV + offset;
+  } else if (sector == 2) {
+    uPrime = moduleV - moduleU;
+    vPrime = -moduleU + offset;
+  } else {
+    throw cms::Exception("RotationException") << "HGCalGeomRotation: desired sector must be eother 0, 1 or 2";
```

Comment:
typo: eother -> other
also, the logic in this block doesn't actually accept a value of 0

Reply:
thanks - I have rearranged the code now.